projects/kit/utils/math/key-steps.ts

```diff
@@ -0,0 +1,48 @@
+import {round} from '@taiga-ui/cdk';
+import {TUI_FLOATING_PRECISION} from '@taiga-ui/kit/constants';
+import {TuiKeySteps} from '@taiga-ui/kit/types';
+
+function tuiFindKeyStepsBoundariesByFn(
```

Comment:
is it worth write unit tests for them without considering component?

Reply:
We have already 92 unit tests + 12 cypress tests to test keySteps (Slider + Range) 🤔 

It is not difficult for me to repeat the same scenarios for functions from

https://github.com/Tinkoff/taiga-ui/blob/e7f3d1bfedf0da6eb38ac16ce6a0761838e373b1/projects/kit/components/slider/test/slider-key-steps.spec.ts#L61-L92 

Should I ?

Reply:
No, but as I understand it, the component still does some transformations

Reply:
> the component still does some transformations

Yes, they does.
I think I can come up some with a few new tests for this function.
For type coverage, at least))

Reply:
I added new test cases for this functions ✅

Reply:
amazing!